examples/hello-world/background.js

```diff
@@ -0,0 +1,21 @@
+// Extension event listeners are a little different from the patterns you may hav seen in DOM or
+// Node.js APIs. Our event listener registration can be broken in to 4 distinct parts:
+//
+// * chrome      - the global namespace for extension APIs.
+// * action      – the namespace of the specific API we want to use.
+// * onClicked   - the event we want to subscribe to.
+// * addListener - what we want to do with this event.
```

Comment:
I'd say "the method which adds the handler" here.

Reply:
The intent of the current copy was to focus on the role of the last parameter. While we are registering the event listener, we could call a number of other methods in this position. The full list of methods is available at https://developer.chrome.com/docs/extensions/reference/events/#type-Event.

Any thoughts on how we can better communicate this intent?

Reply:
I see your point. I think for the vast majority of people this is all they use so it feels a bit superfluous. But I'm not going to argue this point, I think it's fine.